helm/charts/kratos/values.yaml

```diff
@@ -389,6 +389,13 @@ statefulSet:
   #      lines, adjust them as necessary, and remove the curly braces after 'annotations:'.
   #      e.g.  sidecar.istio.io/rewriteAppHTTPProbers: "true"
 
+  # -- The secret specified here will be used to load environment variables with envFrom.
+  # This allows arbitrary environment variables to be provided to the application which is useful for
+  # sensitive values which should not be in a configMap.
+  # This secret is not created by the helm chart and must already exist in the namespace.
+  # https://kubernetes.io/docs/tasks/inject-data-application/distribute-credentials-secure/#configure-all-key-value-pairs-in-a-secret-as-container-environment-variables
+  # environmentSecretsName:
```

Comment:
the value can be kept in the default values as `""` or `null` to make it more verbose. 
Please add a test, in the https://github.com/ory/k8s/tree/master/hacks/manifests dir add an example secret to be loaded and in the https://github.com/ory/k8s/blob/master/hacks/values/kratos.yaml set the value to point to that secret :)

Reply:
hacks/manifests/env-secrets.yaml: https://github.com/ory/k8s/pull/589/files#diff-33a5a5786a496aa6051cdcbbe89bbb8a204fe0ede2f9ade7ba6d5754311735df

hacks/values/kratos.yaml: https://github.com/ory/k8s/pull/589/files#diff-589731bd7b6a30a19645b4259c330b77fc6e5003b66fe9692f13fdef8d09d4cfR237

environmentSecretsName for StatefullSet is added to default values file in the same way as environmentSecretsName for Deployment is present